CHANGES_NEXT_RELEASE

```diff
@@ -1,13 +1 @@
-- Upgrade async dependency from 2.6.1 to 2.6.2
-- Upgrade body-parser dependency from ~1.18.3 to ~1.19.0
-- Upgrade moment dependency from ~2.22.2 to ~2.24.0
-- Upgrade moment-timezone dependency from ~0.5.21 to ~0.5.25
-- Upgrade mongodb dependency from 3.1.8 to 3.2.3
-- Upgrade mongoose dependency from 5.3.6 to 5.5.4
-- Upgrade query-string dependency from 6.2.0 to 6.5.0
-- Upgrade coveralls dev dependency from ~3.0.2 to ~3.0.3
-- Upgrade jshint dev dependency from ~2.9.6 to ~2.10.2
-- Upgrade mocha dev dependency from 5.2.0 to 6.1.4
-- Upgrade nock dev dependency from 10.0.1 to 10.0.6
-- Upgrade timekeeper dev dependency from 2.1.2 to 2.2.0
-- Set Nodejs 8 as minimum version in packages.json (effectively removing Nodev6 from supported versions)
\ No newline at end of file
+Add: /iot/configGroups API endpoints (as equivalent to /iot/services) (#752)
```

Comment:
This time it seems you have deleted too much :)

The rule for CHANGES_NEXT_RELEASE is that a PR has to include only the lines corresponding to the fixes/changes of this PR, the other lines untouched.

The easy solution is:

1. Remove CHANGEX_NEXT_RELEASE file in your working copy in your branch
2. Create it again with the current content in master: https://github.com/telefonicaid/iotagent-node-lib/blob/master/CHANGES_NEXT_RELEASE
3. Add the line 

```
- Add: /iot/configGroups API endpoints (as equivalent to /iot/services) (#752)
```

Then git add & commit & push.

Reply:
Fixed. CHANGES_NEXT_RELEASE looks correct at commit e6cee6a